backend/src/modules/doubles-tournament/presentation/rest-api/DoublesTournamentRouter.ts

```diff
@@ -48,7 +53,33 @@ export function doublesTournamentRouter(
     return response.status(StatusCodes.OK).json(new MatchesQueueDto(queryResult.tournamentId.raw, queuedMatchesDto));
   };
 
+  const getAllTournaments = async (request: Request, response: Response) => {
+    const queryResult = await queryPublisher.execute<FindAllDoublesTournamentsResult>(new FindAllDoublesTournaments());
+    if (!queryResult) {
+      return response.status(StatusCodes.NOT_FOUND).json({ message: "There aren't any tournaments ready to start" });
+    }
+    return response.status(StatusCodes.OK).json(new TournamentListDto(queryResult.map(toTournamentDto)));
+  };
+
+  const startTournament = async (request: Request, response: Response) => {
+    const requestBody: PostTournamentStartRequestBody = request.body;
+    const queryResult = await queryPublisher.execute<FindDoublesTournamentByIdResult>(new FindDoublesTournamentById(requestBody));
+    if (!queryResult) {
+      return response.status(StatusCodes.NOT_FOUND).json({ message: "There aren't any tournaments ready to start" });
+    }
+    if (queryResult.status === 'STARTED') {
+      return response.status(StatusCodes.BAD_REQUEST).json({ message: 'Such tournament was already started' });
+    }
```

Comment:
@nowakprojects czy te if'y są ok? Bo w testach jak próbuję jakieś wywołać to nie mogę wejść do nich i wywala, że jest ok wszystko i wystartowało turniej.

Reply:
Wowowow, paaanie. Wystrzeliłeś logikę domenową (domain) aż do presentation (rest api). Jasne, że są w złym miejscu :)
Chyba już za późno było? 
Wiesz co jest nie tak i jak powinno być?
A to, ze testy nie wchodzą to jasne, bo w testach REST API patrzymy tylko czy one wysyłają właściwe command / query.

Reply:
no to faktycznie trochę poleciałem :-P

Reply:
Wszystko (czyli status) co potrzebujesz dal zaakceptowania / odrzucenia command masz w pobieranym w commandhandlerze DoublesTournament.

Reply:
chociaż ... no już sam nie wiem - fakt było późno, ale to co te dwa if'y związane z queryResult wywalić?

Reply:
wtedy te zakomentowane testy są zbędne, ale wydaje mi się , że w sumei te dwa if'y to nie są jakąś przesadą

Reply:
a.AA dobra ... nie wiem czemu wrzuciłem te ify tutaj jak pierwszego ifa to już mam w obiekcie domenowym zrobionego i drugiego też tutaj wypadałoby mieć
.
.
. 
never mind